$MirOS: ports/www/lynx/patches/patch-src_LYMain_c,v 1.4 2006/10/02 23:22:07 tg Exp $
--- src/LYMain.c.orig	Fri Oct 13 00:14:55 2006
+++ src/LYMain.c	Fri Oct 13 19:39:43 2006
@@ -64,7 +64,7 @@
 #define COOKIE_FILE "cookies"
 #define TRACE_FILE "LY-TRACE.LOG"
 #else
-#define COOKIE_FILE ".lynx_cookies"
+#define COOKIE_FILE ".etc/cookies.txt"
 #define TRACE_FILE "Lynx.trace"
 #endif /* FNAMES_8_3 */
 
@@ -1168,6 +1168,11 @@ int main(int argc,
     StrAllocCopy(LYUserAgent, LYNX_NAME);
     StrAllocCat(LYUserAgent, "/");
     StrAllocCat(LYUserAgent, LYNX_VERSION);
+    /*
+     * rationale: certain web backends only give Mozilla or Internet
+     * Explorer users the "real" web page by checking the user agent
+     */
+    StrAllocCat(LYUserAgent, " (compatible; MSIE 6.0; UNIX)");
     if (HTLibraryVersion) {
 	StrAllocCat(LYUserAgent, " libwww-FM/");
 	StrAllocCat(LYUserAgent, HTLibraryVersion);
@@ -2057,10 +2062,7 @@ int main(int argc,
      * synchronized.  - FM
      */
     if (!bookmark_page || *bookmark_page == '\0') {
-	temp = NULL;
-	HTSprintf0(&temp, "lynx_bookmarks%s", HTML_SUFFIX);
-	set_default_bookmark_page(temp);
-	FREE(temp);
+	set_default_bookmark_page("./.etc/bookmark.htm");
     }
     if (!BookmarkPage || *BookmarkPage == '\0') {
 	set_default_bookmark_page(bookmark_page);
